src/applications/disability-benefits/all-claims/utils/index.jsx

```diff
@@ -684,6 +684,14 @@ export const show526Wizard = state => toggleValues(state).show526Wizard;
 export const showSubform8940And4192 = state =>
   toggleValues(state)[FEATURE_FLAG_NAMES.subform89404192];
 
+/**
+ * Selector to check if the toxic exposure destruction modal feature flag is enabled.
+ * @param {Object} state - Redux state object
+ * @returns {boolean} True if the feature flag is enabled, false otherwise
+ */
+export const showToxicExposureDestructionModal = state =>
```

Comment:
Ah ok so this is just the PR with the function we will use but not where it's used? That makes sense, yeah safer to review and roll out in pieces like this nice